src/common/time/src/timezone.rs

```diff
@@ -52,6 +52,16 @@ pub fn get_timezone(tz: Option<Timezone>) -> Timezone {
     })
 }
 
+#[inline(always)]
+/// If the `tz = Some("") || None || Some(Invalid timezone)`, return system timezone,
+/// or return parsed `tz` as timezone.
+pub fn parse_timezone(tz: Option<&str>) -> Timezone {
```

Comment:
We'd better add a UT for this function as it depends on third party function

Reply:
`parse_timezone` is mainly the wapper of `Timezone::from_tz_string`. In theory we should throw a error when we receive a invalid timezone. But now during the process of parsing the HTTP header, if there is an error in the header, no error will be reported, but the default value will be written directly, which may sometimes confuse users.

Reply:
`Timezone::from_tz_string` is our internal function, it has been tested